src/Server/Templates.fs

```diff
@@ -0,0 +1,41 @@
+module ServerCode.Templates
+
+
+
+open System.Threading.Tasks
+open Microsoft.AspNetCore.Http
+open Giraffe
+open ServerCode.Domain
+open ServerCode.FableJson
+open ServerTypes
+open Client.Shared
+open Giraffe.GiraffeViewEngine
+open Fable.Helpers.ReactServer
+
+let index (model: Model) =
+  let jsonState = toJson (toJson model)
```

Comment:
toJson^2 ?

Reply:
Yeah, I explained this in [the SSR tutorial](https://github.com/fable-compiler/fable-react/blob/master/docs/server-side-rendering.md#step-3-create-your-initial-state-on-the-server-side), I can add more comments here if you think it's needed.